woocommerce/woocommerce-ios
# Product Creation AI: Populate preview view with product details

Closes: #10760


## Description

This PR updates the preview view of the product creation with AI flow to populate data from a `Product` model. Only sample data is currently used for demonstration purposes. Unit tests will be added once we integrate the API into this view model.

## Testing instructions

- Log in to a WPCom or self-hosted store with Jetpack AI installed.
- Switch to the Products tab and select "+" to add a new product.
- Select add product with AI.
- Enter any product name and features to get to the preview screen.
- Notice that the UI displays sample data from [`swiftUIPreviewSample`](https://github.com/woocommerce/woocommerce-ios/blob/bdba4f3fc7003931e2347bb7125336f1af53089e/WooCommerce/Classes/Extensions/Product%2BSwiftUIPreviewHelpers.swift#L7).

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/5533851/a25f37b1-75ba-4fdd-a9f4-9d4e21f1a62b



---
